src/envoy/mixer/grpc_transport.h

```diff
@@ -28,13 +28,31 @@ namespace Envoy {
 namespace Http {
 namespace Mixer {
 
+// A helper class to pass Upstream::ClusterManager& in the lambda capture
+// If passing Upstream::ClusterManager& directly in the lambda capture,
```

Comment:
You are right, have to use & in the lambda capture to pass a reference type. Thanks.
The whole class is removed.